development/webpack/utils/config.ts

```diff
@@ -0,0 +1,186 @@
+import { join } from 'node:path';
+import { readFileSync } from 'node:fs';
+import { parse as parseYaml } from 'yaml';
+import { parse } from 'dotenv';
+import { setEnvironmentVariables } from '../../build/set-environment-variables';
+import type { Variables } from '../../lib/variables';
+import { type Args } from './cli';
+import { getMetaMaskVersion } from './helpers';
+
+const BUILDS_YML_PATH = join(__dirname, '../../../builds.yml');
+
+/**
+ * Coerce `"true"`, `"false"`, and `"null"` to their respective JavaScript values.
+ *
+ * @param value
+ * @returns
+ */
+function coerce(value: string) {
+  if (value === 'true') return true;
+  if (value === 'false') return false;
+  if (value === 'null') return null;
```

Comment:
I _think_ we need to coerce `""` to `undefined` or `null`. Some people have `PHISHING_WARNING_PAGE_URL=` in their .metamaskrc, which I think the gulp build is processed as undefined or null, but treated as `""` in the webpack build.